### ISSUE TITLE
Provide php-xdebug package for PHP 7.3

### ISSUE THREAD

**Issue by markvds** [ByteInternet/hypernode-docker#34]
As PHP 7.3 is supported as of today (thanks for that!), it would be nice if xdebug would also be available for installation. Unfortunately I'm not comfortable enough with `apt` to fully understand why it's not available - whether it's not available at all or only in this specific `byte2` version.

`apt-cache` says about the `php-xdebug` package (version _2.6.0+2.5.5-byte2_):
> Provides: php5.5-xdebug, php5.6-xdebug, php7.0-xdebug, php7.1-xdebug, php7.2-xdebug

TL;DR It would be nice to have `php-xdebug` for PHP 7.3 available to install

**Comment by vdloo** [comment]
Hi @markvds, yeah I'm aware. I built a new version of [php-xdebug](https://salsa.debian.org/php-team/pecl/xdebug) but found that PHP 5.5 and PHP 5.6 are dropped (and we still support those). I have to figure out figure out something for this some time but skipped that for now because we don't install php-xdebug on production Hypernodes (as that would interfere with our monitoring among other things).

after:
```
root@3502dc-phpxdebug-magweb-vgr /vagrant # dpkg -L php-xdebug 
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/php-xdebug
/usr/share/doc/php-xdebug/copyright
/usr/share/doc/php-xdebug/changelog.Debian.gz
/usr/lib
/usr/lib/php
/usr/lib/php/20180731
/usr/lib/php/20180731/xdebug.so
/usr/lib/php/20170718
/usr/lib/php/20170718/xdebug.so
/usr/lib/php/20151012
/usr/lib/php/20151012/xdebug.so
/usr/lib/php/20160303
/usr/lib/php/20160303/xdebug.so
/etc
/etc/php
/etc/php/7.3
/etc/php/7.3/mods-available
/etc/php/7.3/mods-available/xdebug.ini
/etc/php/7.0
/etc/php/7.0/mods-available
/etc/php/7.0/mods-available/xdebug.ini
/etc/php/7.2
/etc/php/7.2/mods-available
/etc/php/7.2/mods-available/xdebug.ini
/etc/php/7.1
/etc/php/7.1/mods-available
/etc/php/7.1/mods-available/xdebug.ini
```

before:
```
root@3502dc-phpxdebug-magweb-vgr /vagrant # dpkg -L php-xdebug
/.
/etc
/etc/php
/etc/php/5.5
/etc/php/5.5/mods-available
/etc/php/5.5/mods-available/xdebug.ini
/etc/php/5.6
/etc/php/5.6/mods-available
/etc/php/5.6/mods-available/xdebug.ini
/etc/php/7.0
/etc/php/7.0/mods-available
/etc/php/7.0/mods-available/xdebug.ini
/etc/php/7.1
/etc/php/7.1/mods-available
/etc/php/7.1/mods-available/xdebug.ini
/etc/php/7.2
/etc/php/7.2/mods-available
/etc/php/7.2/mods-available/xdebug.ini
/usr
/usr/lib
/usr/lib/php
/usr/lib/php/20170718
/usr/lib/php/20170718/xdebug.so
/usr/lib/php/20131226
/usr/lib/php/20131226/xdebug.so
/usr/lib/php/20160303
/usr/lib/php/20160303/xdebug.so
/usr/lib/php/20121212
/usr/lib/php/20121212/xdebug.so
/usr/lib/php/20151012
/usr/lib/php/20151012/xdebug.so
/usr/share
/usr/share/doc
/usr/share/doc/php-xdebug
/usr/share/doc/php-xdebug/changelog.Debian.gz
/usr/share/doc/php-xdebug/copyright
/etc/php/7.3/mods-available/xdebug.ini
```

I did put this deb somewhere for you to download if you want:
```
# wget sharkdance.hypernode.io/php-xdebug_2.7.0-byte3_amd64.deb
# dpkg -i php-xdebug_2.7.0-byte3_amd64.deb
```
but note that this link is volatile and will probably die some time soon. I'll see if I can find some time for a nicer solution soon but at least for now you won't have to be blocked on this.

**Comment by vdloo** [comment]
@markvds this should now work as before also for PHP 7.3: https://support.hypernode.com/changelog/release-6591-php-xdebug-for-php-7-3-for-hypernode-docker/

**Comment by markvds** [comment]
@vdloo Did I say thank you? No I didn't!

Thank you for the quick response and quick resolution! Works like a charm!
